.github/workflows/ci-workflow.yml

```diff
@@ -18,13 +18,33 @@ jobs:
 
       - run: sudo npm i -g ts-node
 
-      - name: Test
+      - name: Resource group scope test
         env:
           INPUT_SCOPE: resourcegroup
           INPUT_SUBSCRIPTIONID: ${{ secrets.SUBSCRIPTION_ID }}
-          INPUT_RESOURCEGROUPNAME: azurearmaction
+          INPUT_RESOURCEGROUPNAME: E2eTestResourceGroupForArmAction
+          INPUT_TEMPLATE: https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-webapp-basic-linux/azuredeploy.json
+          INPUT_PARAMETERS: https://github.com/Azure/azure-quickstart-templates/blob/master/101-webapp-basic-linux/azuredeploy.parameters.json
+          INPUT_DEPLOYMENTNAME: github-test-rg
+          INPUT_DEPLOYMENTMODE: Incremental
```

Comment:
I think default value for mode is `Incremental`. Do you want change/add one test to use `Complete` mode